src/ert/plugins/hook_implementations/forward_model_steps.py

```diff
@@ -422,6 +422,39 @@ def documentation() -> Optional[ForwardModelStepDocumentation]:
         )
 
 
+class MoveDirectory(ForwardModelStepPlugin):
+    def __init__(self) -> None:
+        super().__init__(
+            name="MOVE_DIRECTORY",
+            command=[
+                str(
+                    (
+                        Path(__file__)
+                        / "../../../resources/shell_scripts/move_directory.py"
+                    ).resolve()
+                ),
+                "<FROM>",
+                "<TO>",
+            ],
+        )
+
+    @staticmethod
+    def documentation() -> Optional[ForwardModelStepDocumentation]:
+        return ForwardModelStepDocumentation(
+            category="utility.file_system",
+            description="""
+The :code:`MOVE_DIRECTORY` job will move directory to target directory.
+If directory already exists, this job will move directory to the target directory
```

Comment:
Would write something more like:
If <TO> directory already exists, it will be replaced.